Grocery_List_Zhecan_Wang/app/src/main/java/com/example/jong/todolist/MainActivity.java

```diff
@@ -0,0 +1,129 @@
+package com.example.jong.todolist;
+
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.EditText;
+import android.widget.ListView;
+
+import java.util.ArrayList;
+
+public class MainActivity extends AppCompatActivity {
+    
+    ArrayList<String> arraylist = new ArrayList();
```

Comment:
Really nitpicking here, but this is an unchecked assignment which in some professional code environments is not approved. 
This explains more: http://stackoverflow.com/questions/2693180/what-is-unchecked-cast-and-how-do-i-check-it
You can avoid that by doing
```
ArrayList<String> myList = new ArrayList<String>();
```